test/test_helper.exs

```diff
@@ -2,28 +2,28 @@ ExUnit.start()
 
 # Mostly taken from here https://elixirforum.com/t/generating-5-000-random-strings-improving-performance-by-an-order-of-magnitude-but-can-it-be-better/23206/5
 
-  defmodule Rando do
-    chars = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ123456789'
+defmodule Rando do
```

Comment:
This might be useful: https://github.com/thoughtbot/ex_machina

Reply:
This looks cool, but it seems a bit overkill for the current use case.